### PR TITLE
svcat: Show ParametersFrom in describe commands

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: cmd/svcat/output/yaml.go [hunk 1]
```diff
@@ -59,3 +60,16 @@ func writeParameters(w io.Writer, parameters *runtime.RawExtension) {
 		writeYAML(w, params, 2)
 	}
 }
+
+func writeParametersFrom(w io.Writer, parametersFrom []v1beta1.ParametersFromSource) {
+	if parametersFrom == nil || len(parametersFrom) == 0 {
```
Comment (jpeeler) [inline cmd/svcat/output/yaml.go]: Type safety combined with passing by value means that the first part of the if is unnecessary.

File: cmd/svcat/output/yaml.go [hunk 2]
```diff
@@ -59,3 +60,16 @@ func writeParameters(w io.Writer, parameters *runtime.RawExtension) {
...
+		return
+	}
+
+	fmt.Fprintln(w, "\nParameters From:")
+	for _, p := range parametersFrom {
+		fmt.Fprintln(w, "  Secret Key Ref:")
+		fmt.Fprintf(w, "    Name: %s\n", p.SecretKeyRef.Name)
```
Comment (jpeeler) [inline cmd/svcat/output/yaml.go]: Somewhat opposite to above, SecretKeyRef could be nil here so check for that before attempting to print. (This also means that potentially there may be no output, yet the "Parameters From:" header would still get printed, which would look weird.)

Comment (luksa) [reply cmd/svcat/output/yaml.go]: Thanks. I missed the fact that `SecretKeyRef`is optional.

File: cmd/svcat/output/yaml.go [hunk 3]
```diff
@@ -59,3 +60,22 @@ func writeParameters(w io.Writer, parameters *runtime.RawExtension) {
...
+	}
+
+	headerPrinted := false
+	for _, p := range parametersFrom {
+		if p.SecretKeyRef != nil {
+			if !headerPrinted {
+				fmt.Fprintln(w, "\nParameters From:")
+				headerPrinted = true
+			}
+			fmt.Fprintln(w, "  Secret Key Ref:")
```
[the remark this answers is not in the document]
Comment (luksa) [reply cmd/svcat/output/yaml.go]: Your wish is my command. :)  Done.

[the remark this answers is not in the document]
Comment (carolynvs) [reply cmd/svcat/output/yaml.go]: Careful! I'll start getting delusions on grandeur. 😉

**Pull request merged**